consensus/integration/signer_test.go

```diff
@@ -26,15 +26,12 @@ func (s *Signer) CreateVote(block *model.Block) (*model.Vote, error) {
 	return vote, nil
 }
 func (*Signer) CreateQC(votes []*model.Vote) (*flow.QuorumCertificate, error) {
-	voterIDs := make([]flow.Identifier, 0, len(votes))
-	for _, vote := range votes {
-		voterIDs = append(voterIDs, vote.SignerID)
-	}
 	qc := &flow.QuorumCertificate{
-		View:      votes[0].View,
-		BlockID:   votes[0].BlockID,
-		SignerIDs: voterIDs,
-		SigData:   nil,
+		View:    votes[0].View,
+		BlockID: votes[0].BlockID,
+		// TODO: fix
+		SignerIndices: nil,
```

Comment:
https://github.com/onflow/flow-go/pull/2140/files#diff-0651c91205a7f0e131c435303abb380ea71331d7149406658db82852a8235174
This doesn't need to be fixed actually, because the entire signature validation is mocked away